src/BundlerMinifier/MSBuild/BuildBundlerMinifier.targets

```diff
@@ -11,7 +11,7 @@
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerBuildTask"/>
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerCleanTask"/>
 
-  <Target Name="BundleMinify" Condition="'$(RunBundleMinify)' != 'False'">
+  <Target Name="BundleMinify" DependsOnTargets="CompileTypeScript" BeforeTargets="AssignTargetPaths" Condition="'$(RunBundleMinify)' != 'False'">
```

Comment:
Instead of `DependsOnTargets`, couldn't it use `AfterTargets`? That would make sure it works even if people don't use TypeScript

Reply:
@304NotModified I think you should remove the `BuildDependsOn` property. After that the `AfterTargets` and `BeforeTargets` will work more reliable.
```xml
<Target Name="BundleMinify" AfterTargets="CompileTypeScriptWithTSConfig" BeforeTargets="BeforeCompile" Condition="'$(RunBundleMinify)' != 'False'">
```
- It will always run before the `BeforeCompile` target.
- And if TypeScript is used in the project, it will run after the `CompileTypeScriptWithTSConfig` target. It's better to use this target instead of the `CompileTypeScript`.

Will you update the targets or should I send a new PR?